alpine-consul/Dockerfile

```diff
@@ -18,7 +18,15 @@ RUN export GOPATH=/go && \
     addgroup consul && \
     adduser -D -G consul consul && \
     mkdir -p /data/consul && \
-    chown -R consul:consul /data/consul
+    chown -R consul:consul /data/consul    
+
+# Download and install Consul Template
```

Comment:
@matthewvalimaki, do you think it would be better to have `consul-template` only `alpine-consul-base` only, not on `alpine-consul`. The theory is if you want to run consul agents in server mode you use `alpine-consul`. If you want to run a service (and therefore consul agents in agent mode) you use `alpine-consul-base`. The question is, is there an instance in which you'd need `consul-template` while running consul in server mode?

Sorry to ask such a question this late in the PR. If you agree but don't want to make the change, I can do it when I merge...